inst/rmarkdown/templates/template_sfp_en/skeleton/skeleton.Rmd

```diff
@@ -10,6 +10,9 @@ params:
   version_number: "{{{version_number}}}"
   language: {{{language}}}
   theme: {{{theme}}}
+  dependencies_protocolcode: [sfp_4##_en, sfp_3##_en]
+  dependencies_versionnumber: [YYYY.NN, YYYY.NN]
+  dependencies_params: !r list(list(width = 5, length = 5), list())
```

Comment:
@hansvancalster I think you mentioned the idea of defining which dependencies to render in an appendix (hence regard as 'subprotocol'), and which ones not. If that view is still valid, then you probably need another `dependencies_xxx` parameter to capture that (an array of booleans?).

Reply:
Yep, that needs to be added.

Reply:
done in e33fbe3